tests/testfiles/CommonMark_0.30/fenced_code_blocks/025.test

```diff
@@ -18,6 +18,6 @@ end
 documentBegin
 BEGIN fencedCode
 - src: ./_markdown_test/bec320a1e3b741ba35aaee15586aa06d.verbatim
-- infostring: ruby startline=3 (dollarSign)(percentSign)@(hash)(dollarSign)
+- infostring: ruby startline=3 $%@#$
```

Comment:
Since infostring are programming strings that are not expected to enter the typographic material that TeX typesets, we don't escape special characters beside the left brace (`{`) and the right brace (`}`).